apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.ts

```diff
@@ -144,6 +148,9 @@ export class ListManufacturingComponent implements OnInit {
         this.currentOrganizationType = this.generalService.currentOrganizationType;
         this.getWarehouses();
 
+        this.manufacturingSearchRequest.count = this.paginationLimit;
+        this.manufacturingSearchRequest.page = this.currentPage;
```

Comment:
Confirmed with Ravinder sir

Reply:
I have checked it is will work same in test